grails-app/migrations/views/cycle-count-session.sql

```diff
@@ -1,40 +1,56 @@
-CREATE OR REPLACE VIEW cycle_count_session AS (
-	SELECT
-		# ID hash of product.id and location.id
-		CRC32(CONCAT(location.id, product.id)) as id,
-        inventory.id as inventory_id,
-		product.id as product_id,
-		location.id as facility_id,
-        cycle_count_request.id as cycle_count_request_id,
-        COALESCE(inventory_level.abc_class, product.abc_class) as abc_class,
+CREATE OR REPLACE VIEW cycle_count_session AS
+(
+SELECT
+    # ID hash of product.id and location.id
+    CRC32(CONCAT(location.id, product.id))                                       as id,
+    inventory.id                                                                 as inventory_id,
+    product.id                                                                   as product_id,
+    location.id                                                                  as facility_id,
+    #cycle_count_request.id as cycle_count_request_id,
+    NULL                                                                         as cycle_count_request_id,
+    COALESCE(inventory_level_summary.abc_class, product.abc_class)                       as abc_class,
 
-        # Consolidate the statuses down to a single field representing the status of the candidate itself.
-        COALESCE(cycle_count.status, cycle_count_request.status) as status,
+    # Consolidate the statuses down to a single field representing the status of the candidate itself.
+    NULL                                                                         as status, # COALESCE(cycle_count.status, cycle_count_request.status) as status,
```

Comment:
By the way, this is coming back. I'm just trying to figure out best way to pull it in.

Reply:
And it's back.